app/assets/stylesheets/sessions.scss

```diff
@@ -1,3 +1,30 @@
-// Place all the styles related to the sessions controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+body {
+    font-family: 'Lato', sans-serif;
+
+  #login{
+    text-align: center;
+  }
+
+  a{
+    display: inline-block;
+    background-color: #41B6A6;
+    color: white;
+    margin-left: 30px;
```

Comment:
i see you have this in all your scss.... if you need this in every place... might as well but it in application.scss